spec/system/kudos_spec.rb

```diff
@@ -7,7 +7,9 @@
 
   let(:giver) { create(:employee) }
   let!(:receiver) { create(:employee) }
-  let!(:kudo) { create(:kudo, giver: giver, receiver: receiver) }
+  let!(:company_value) { create(:company_value) }
+  let!(:company_value2) { create(:company_value, title: 'Second Company Value') }
+  let!(:kudo) { create(:kudo, giver: giver, receiver: receiver, company_value: company_value) }
```

Comment:
FYI you can write like this, an your factory will create a kudo with a default company_value

```suggestion
  let!(:kudo) { create(:kudo, giver: giver, receiver: receiver,) }
```